packages/aila/src/features/americanisms/AilaAmericanisms.ts

```diff
@@ -8,17 +8,20 @@ import type {
   AmericanismIssue,
   AmericanismIssueBySection,
 } from "../../features/americanisms";
-import type { LessonPlanKey, LooseLessonPlan } from "../../protocol/schema";
 
 export type TranslationResult = Record<
   string,
   Array<{ [phrase: string]: { issue: string; details: string } }>
 >;
 
-export class AilaAmericanisms implements AilaAmericanismsFeature {
+export type AilaDocumentContent = Record<string, unknown>;
+
+export class AilaAmericanisms<T extends AilaDocumentContent>
```

Comment:
This removes the association with the lesson plan and treats it as an unknown object.